VenafiPS/Public/Get-TppIdentity.ps1

```diff
@@ -119,6 +134,12 @@ function Get-TppIdentity {
                         $response | Add-Member @{ 'Associated' = $associated.Identities }
                     }
 
+                    if (($response.IsGroup) -and ($GetMembers))  {
+                        $params.UriLeaf = 'Identity/GetMembers'
+                        $params.Body.Add("ResolveNested","1");
```

Comment:
do we always want to do this or should it be a param?
also...
```suggestion
                        $params.Body.ResolveNested = '1'
```

Reply:
I'm torn on this one.  I would like to  keep this simple as possible for testing and to implement asap.  We could always add a defaulted parameter later on, I have  yet  to see a  use case to not set ResolveNested .